tests/unit/core/core.js

```diff
@@ -138,4 +138,89 @@ test( "uniqueId / removeUniqueId", function() {
 	equal( el.attr( "id" ), null, "unique id has been removed from element" );
 });
 
+// Support: Core 1.9 Only, IE8 Only
+// The use of $.trim() in the two tests below this is to account for IE8 missing string.trim()
```

Comment:
The support comment should be directly above every place that support is needed.

Uses of `$.trim()` don't need support comments.